### PR TITLE
Check for null geometries

### PR DESCRIPTION
**Pull request by kad-korpem**
# Description

Add check on null geometries to RQ24.

## Type of change

(Remove irrelevant options)

- Improvement of existing feature

# Checklist:

- [x] I've double-checked the code in this PR myself
- [x] I've left the code better than before ([boy scout rule](https://wiki.c2.com/?BoyScoutRule))
- [x] The code is readable, comments are added that explain hard or non-obvious parts.
- [x] I've expanded/improved the (unit) tests, when applicable
- [x] I've run (unit) tests that prove my solution works
- [x] There's no sensitive information like credentials in my PR

### CODE REVIEW

File: tests/validations/test_geometry_valid_check.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def test_with_gpkg_valid_simple():
     assert checks[0][1] == "geometry"
     assert checks[0][2] == "Self-intersection"
     assert checks[0][3] == 1
-    assert checks[0][4] == 1
+    # assert checks[0][4] == 1
```
Comment (roelarents) [inline tests/validations/test_geometry_valid_check.py:39]: waarom uitgecomment? misschien gewoon weg anders

Comment (kad-korpem) [reply tests/validations/test_geometry_valid_check.py:39]: Ho, foutje van testen overgebleven 🙈

File: tests/validations/test_geometry_empty_check.py [hunk 2]
```diff
@@ -10,7 +10,18 @@ def test_with_gpkg_empty():
     assert len(result) == 1
     assert (
         result[0]
-        == "Found empty geometry in table: stations, column geom, 45 times, example id 129"
+        == "Found empty geometry in table: test_geometry_empty, column geom, 45 times, example id 129"
+    )
+
+
+def test_with_gpkg_null():
+    # geometries that are null slip through
```
Comment (roelarents) [inline tests/validations/test_geometry_empty_check.py:18]: not anymore now, right?

Comment (kad-korpem) [reply tests/validations/test_geometry_empty_check.py]: Fixed

**Review by roelarents — APPROVED** [review]

**Pull request merged**